Modules/_abc.c

```diff
@@ -629,15 +623,16 @@ _abc__abc_subclasscheck_impl(PyObject *module, PyObject *self,
         goto end;
     }
 
+    _abcmodule_state *state = get_abc_state(module);
```

Comment:
Some compilers don't like variable definitions in the middle of a code block. Please define the variable at the top but keep the assignment here.